ql/src/java/org/apache/hadoop/hive/ql/ddl/catalog/desc/DescCatalogDesc.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.ql.ddl.catalog.desc;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.ql.ddl.DDLDesc;
+import org.apache.hadoop.hive.ql.plan.Explain;
+
+import java.io.Serializable;
+
+/**
+ * DDL task description for DESC CATALOG commands.
+ */
+@Explain(displayName = "Describe Catalog", explainLevels = { Explain.Level.USER, Explain.Level.DEFAULT, Explain.Level.EXTENDED })
+public class DescCatalogDesc implements DDLDesc, Serializable {
+  private static final long serialVersionUID = 1L;
+
+  public static final String DESC_CATALOG_SCHEMA = "cat_name,comment,location#string:string:string";
+
+  public static final String DESC_CATALOG_SCHEMA_EXTENDED = "cat_name,comment,location,create_time#string:string:string:string";
```

Comment:
What's the usage of this code?

Reply:
The schema is used in `createFetchTask`.